app/Http/Controllers/ApiController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Clans;
+use Illuminate\Http\Request;
+use Illuminate\Support\Collection;
+
+class ApiController extends Controller
+{
+    private $clans;
+
+    public function __construct()
+    {
+        $this->clans = new Clans();
+    }
+
+    public function getClans(Request $request): Collection
+    {
+        $length = $request->input("length", 10);
+        $sortBy = $request->input("sortBy", "KDR");
+        $order = $request->input("order", "asc") == "asc";
```

Comment:
?

Reply:
`$request->input("order", "asc") == "asc";`
This one for what?

Reply:
For specifying the order.
Ascending or descending.